invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -311,18 +267,88 @@ export class RDMDepositForm extends Component {
                 <VersionField />
                 <PublisherField />
               </AccordionField>
-              {/**TODO: uncomment to use FundingField*/}
-              {/* <AccordionField
-              active={true}
-              label={"Funding"}
+
+              <AccordionField
+                fieldPath=""
+                active={true}
+                label={"Funding"}
+                ui={this.accordionStyle}
               >
-              <FundingField options={this.vocabularies.metadata.funding} />
-              <ComingSoonField
-                fieldPath="metadata.funding"
-                label="Awards"
-                labelIcon="money bill alternate outline"
-              />
-              </AccordionField> */}
+                <FundingField
+                  fieldPath="metadata.funding"
+                  searchConfig={{
+                    searchApi: {
+                      axios: {
+                        headers: {
+                          //  FIXME use for internationalisation
+                          //  Accept: "application/vnd.inveniordm.v1+json"
+                          Accept: "application/json",
+                        },
+                        url: "/api/awards",
+                        withCredentials: false,
+                      },
+                    },
+                    initialQueryState: {
+                      sortBy: "bestmatch",
+                      sortOrder: "asc",
+                      layout: "list",
+                      page: 1,
+                      size: 5,
+                    },
+                  }}
+                  label="Awards"
+                  labelIcon="money bill alternate outline"
+                  deserializeAward={(award) => {
+                    return {
+                      title: award.title.en ?? award.title,
```

Comment:
does it make sense to specify separate deserializer function? at least for readability

Reply:
I can do this. However, my main reason to leave it where it is now was that defining a function would make the code of said function go around line 90 then pass it to the component in line 300, which does not help readability at all (my biased opinion) and in the end it is "just" defining a dict to be returned.